docs/hub-tutorials/join-mainnet.md

```diff
@@ -250,24 +377,25 @@ Environment="DAEMON_ALLOW_DOWNLOAD_BINARIES=false"
 Environment="DAEMON_RESTART_AFTER_UPGRADE=true"
 ```
 
-after the `LimitNOFILE` line and replace `$(which gaiad)` with `$(which cosmovisor)`.
+After the `LimitNOFILE` line and replace `$(which gaiad)` with `$(which cosmovisor)`.
 
-Then setup the daemon
+Run the following to setup the daemon:
 
 ```bash
 sudo -S systemctl daemon-reload
-sudo -S systemctl enable gaiad
+sudo -S systemctl enable <service name>
 ```
 
-We can then start the process and confirm that it is running
+Then start the process and confirm that it's running.
 
 ```bash
-sudo -S systemctl start gaiad
+sudo -S systemctl start <service name>
 
-sudo service gaiad status
+sudo service <service name> status
 ```
 
-## Export State
+
+## Exporting State
 
 Gaia can dump the entire application state into a JSON file. This application state dump is useful for manual analysis and can also be used as the genesis file of a new network.
```

Comment:
Should mention that the node can't be running when exporting state otherwise you can expect a 'resource temporarily unavailable' error.

Reply:
👍